src/generator/css/base.less

```diff
@@ -659,6 +863,237 @@ body .content .hero-banner {
+    @media (min-width: 820px) {
+        grid-template-rows: repeat(3, 1fr); 
+    }
+
+    @media (max-width: 820px) {
+        grid-template-rows: repeat(3, 1fr);
+        @apply gap-y-8;
+    }
+
+    @media (min-width: 430px) and (max-width: 640px) {
+        @apply gap-y-20;
+    }
+    @media (min-width: 520px) and (max-width: 820px) {
+        @apply gap-y-28;
+    }
+  }
+
+  .grid > :nth-child(1),
+  .grid > :nth-child(2) {
+    grid-row: 1;
+  }
+
+  .grid > :nth-child(3),
+  .grid > :nth-child(4) {
+    grid-row: 2;
+  }
+
+  .grid > :nth-child(5),
+  .grid > :nth-child(6) {
+    grid-row: 3;
+  }
+
+  .grid > :nth-child(2),
+  .grid > :nth-child(3),
+  .grid > :nth-child(6),
+  .grid > :nth-child(7) {
+    @apply z-10 my-8 px-6 sm:px-12 pt-16 pb-20;
+    background-color: var(--color-white-primary);
+    align-self: center;
+
+    p {
+      @apply text-sm md:text-xl tracking-normal;
+      line-height: 1.7rem;
+      color: var(--color-grey-primary);
+      white-space: unset;
+      font-weight: 400;
+
+      strong {
+        @apply tracking-widest text-base md:text-4xl mb-4 sm:mb-9 block;
+        font-family: var(--font-h1);
+        font-weight: 700;
+      }
+
+      @media (max-width: 700px) {
+       padding-top: 1rem;
+       padding-bottom: 0;
+      }
+    }
+  }
+
+  /** text boxes **/
+  .grid > :nth-child(2),
+  .grid > :nth-child(6) {
+    grid-column: 7 / end;
+    @apply mx-6 sm:mr-11 sm:ml-0;
+
+    @media (max-width: 820px) {
+        grid-column: 1 / 13;
+        padding-bottom: 1rem;
+        padding-top: 0;
+        /* margin-left: 0; */
+    }
+  }
+
+  .grid > :nth-child(3),
+  .grid > :nth-child(7) {
+    grid-column: 1 / 7;
+    @apply mx-6 sm:ml-11 sm:mr-0;
+
+    @media (max-width: 820px) {
+        grid-column: 1 / 13;
+        padding-bottom: 1rem;
+        padding-top: 0;
+        /* margin-left: 0; */
+    }
+  }
+
+  /** image boxes **/
+  .grid > :nth-child(1),
+  .grid > :nth-child(5) {
+    @apply -my-6 mr-8 p-0;
+    grid-column: 1 / 8;
+
+    @media (max-width: 820px) {
+        margin-bottom: 0;
+        margin-right: 0;
+        margin-top: -10rem;
+        margin-left: 0;
+        grid-column: 1 / 9;
+    }
+  }
+
+  .grid > :nth-child(4),
+  .grid > :nth-child(8) {
+    @apply -my-6 ml-8 p-0;
+    grid-column: 6 / end;
+
+    @media (max-width: 820px) {
+        margin-bottom: 0;
+        margin-left: 0;
+        margin-top: -10rem;
+        margin-right: 0;
+    }
+  }
+
+  .item {
+    @apply relative;
+    z-index: 1;
+
+    img {
+      width: 100%;
+    }
+  }
+}
+
+body .content > section.why-us {
+  @apply relative z-20 -mt-48 pt-8 sm:pt-32 mb-16 sm:mb-36 pb-5 sm:pb-12 ;
+  background-color: var(--color-section-bg);
+
+   .intro.row {
+     padding-bottom: 0;
+     margin-left: -8px;
+
+     @media (min-width: 1400px) {
+        margin-left: 10px;
+     }
+
+    :nth-child(1) {
+      padding-top: 0.5rem;
+    }
+
+     @media (max-width: 640px) {
+        padding-top: 0;
+     }
```

Comment:
Two things:
- Consider using tailwind declarations
- Keep styles at the top of the element. right now there is child styles between this declaration and the declaration of parent styles. This can make the code harder to read and maintain.